app/src/main/java/com/omelan/cofi/model/Step.kt

```diff
@@ -44,6 +44,11 @@ enum class StepType {
 
     abstract val color: Color
     abstract val stringRes: Int
+    fun isNotWaitStepType(): Boolean = listOf(
+        WATER,
+        ADD_COFFEE,
+        OTHER,
+    ).contains(this)
```

Comment:
`isNotWaitStepType` suggest that it checks if step isn't Wait
I would suggest 
```suggestion
    fun isNotWaitStepType(): Boolean = this != WAIT
```

or

```suggestion
    fun isStepWithValue(): Boolean = listOf(
        WATER,
        ADD_COFFEE,
        OTHER,
    ).contains(this)
```

Reply:
Good catch!